src/explorer/mod.rs

```diff
@@ -169,6 +173,28 @@ impl<'a> GitExplorer {
                 }
                 // let branches = branches.map(|b| BranchData::new(b)).collect();
                 self.paint_commit_track(self.repo.head().unwrap().peel_to_commit().unwrap(), branches)
+
+                /*
```

Comment:
These are all the attempts I made to fix the error[0502] I've got when adding the `abort` and `limit_stack` attributes to `GitExplorer` struct.